srv/isc-agent/plugins/tcp/http/models.py

```diff
@@ -100,3 +101,22 @@ def prepare_database():
             signatures.append(Signature(**signature_schema.dict()))
         settings.DATABASE_SESSION.add_all(signatures)
         settings.DATABASE_SESSION.commit()
+
+
+def read_db_and_log():
+    for instance in settings.DATABASE_SESSION.query(RequestLog).order_by(RequestLog.id):
+        log_data = {
```

Comment:
This doesn't appear to format the logs in a way ISC is expecting. Also remember after sending the logs, the table should be emptied